crates/core/Cargo.toml

```diff
@@ -38,6 +37,9 @@ sysinfo = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true, features = ["rt-multi-thread"] }
 
+[target.'cfg(not(windows))'.dependencies]
```

Comment:
Its readme suggests using `[target.'cfg(not(target_env = "msvc"))'.dependencies]` and maybe add the same attribute where it's used.

Reply:
```suggestion
[target.'cfg(not(target_env = "msvc"))'.dependencies]
```